GitUI/UserControls/RevisionGrid.cs

```diff
@@ -1056,30 +1056,27 @@ private class RevisionGridInMemFilter : RevisionGraphInMemFilter
 
             public RevisionGridInMemFilter(string authorFilter, string committerFilter, string messageFilter, bool ignoreCase)
             {
-                SetUpVars(authorFilter, ref _authorFilter, ref _authorFilterRegex, ignoreCase);
-                SetUpVars(committerFilter, ref _committerFilter, ref _committerFilterRegex, ignoreCase);
-                SetUpVars(messageFilter, ref _messageFilter, ref _messageFilterRegex, ignoreCase);
+                SetUpVars(authorFilter, out _authorFilter, out _authorFilterRegex, ignoreCase);
+                SetUpVars(committerFilter, out _committerFilter, out _committerFilterRegex, ignoreCase);
+                SetUpVars(messageFilter, out _messageFilter, out _messageFilterRegex, ignoreCase);
                 if (!string.IsNullOrEmpty(_messageFilter) && MessageFilterCouldBeSHA(_messageFilter))
                 {
-                    SetUpVars(messageFilter, ref _shaFilter, ref _shaFilterRegex, false);
+                    SetUpVars(messageFilter, out _shaFilter, out _shaFilterRegex, false);
                 }
             }
 
             private static void SetUpVars(string filterValue,
-                                   ref string filterStr,
-                                   ref Regex filterRegEx,
+                                   out string filterStr,
+                                   out Regex filterRegEx,
                                    bool ignoreCase)
             {
-                RegexOptions opts = RegexOptions.None;
-                if (ignoreCase)
-                {
-                    opts = opts | RegexOptions.IgnoreCase;
-                }
-
                 filterStr = filterValue?.Trim() ?? string.Empty;
+
+                var options = ignoreCase ? RegexOptions.IgnoreCase : RegexOptions.None;
```

Comment:
:bulb: If you're already moving this, may as well move it all the way inside the `try` block

Reply:
Can do, but there's no way it can throw, right? The code as it stands makes it completely clear which expression is being protected.